### DISCUSSION TITLE
Unity SDK

### DISCUSSION THREAD

**Discussion by ravenzachary** [supabase/supabase#6706] [Feature Requests]
There's a Supabase C# SDK created by the community (https://github.com/supabase-community/supabase-csharp). It would be great to see support for Unity and mobile deployments (iOS and Android) using Unity as part of this effort.

**Comment by Gauravzdre** [comment]
This wrapper for community created c# sdk for unity (https://github.com/kamyker/supabase-unity) works for android but does not work for IOS maybe some platform specific .dll  issue.
